core/services/workflows/v2/config.go

```diff
@@ -27,6 +27,7 @@ type BillingClient interface {
 type EngineConfig struct {
 	Lggr            logger.Logger
 	Module          host.ModuleV2
+	Config          []byte
```

Comment:
Can we call it WASMConfig or ModuleConfig? So it's not confused with Engine config like limits etc.